### PR TITLE
Update auth.rst

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

----

**Related issue :**

### CODE REVIEW

File: source/auth.rst [hunk 1]
```diff
@@ -3,8 +3,8 @@
 .. _`FedCloud Check-in client`: https://aai.egi.eu/fedcloud
 .. _`AppDB`: https://appdb.egi.eu
 
-Authentication
-==============
+Gettin Access
```
Comment (gwarf) [inline source/auth.rst]: Gettin is probably not what should be used.

**Review by gwarf — CHANGES_REQUESTED** [review]

File: source/auth.rst (hunk 1 above)
Comment (enolfc) [reply source/auth.rst]: Slang :) doing thing fast has issues

Comment (enolfc) [inline source/auth.rst]:
```suggestion
How to get access
```

**Review by enolfc — COMMENTED** [review]
Trying to do everything via github web...

File: source/auth.rst [hunk 2]
```diff
@@ -3,8 +3,8 @@
 .. _`FedCloud Check-in client`: https://aai.egi.eu/fedcloud
 .. _`AppDB`: https://appdb.egi.eu
 
-Authentication
-==============
+How to get access
+=============
```
Comment (enolfc) [inline source/auth.rst]:
```suggestion
=================
```

**Review by enolfc — COMMENTED** [review]
why I'm not in my terminal !?

**Comment by enolfc** [comment]
@gwarf  does it look better?

File: source/auth.rst (hunk 1 above)
Comment (gwarf) [reply source/auth.rst]: :)

**Review by gwarf — APPROVED** [review]

**Pull request merged**